core/org.wso2.carbon.base/src/test/java/org/wso2/carbon/base/UnloadTenantTaskImpl.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.base;
+
+/**
+ * Dummy implementation of UnloadTenantTask for testing purpose.
+ */
+public class UnloadTenantTaskImpl implements UnloadTenantTask {
+
+    @Override
+    public void register(int tenantId, Object registration) {
+
```

Comment:
Shall we add a log stating the registration of the tenant? I believe it would be easy to debug if an error occurs